### PR TITLE
Telewave

### PR DESCRIPTION
**Pull request by vsporeddy**
Implementation of official Wavelength rules

### CODE REVIEW

File: games/telewave/telewave_config.json [hunk 1]
```diff
@@ -0,0 +1,266 @@
+{
+    "max_score": 5,
+    "timeout": 3,
+    "wait_time": 10,
+    "team_names": [
+        ["TSM", "CLG"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:6]: ded rivalry

File: games/telewave/telewave.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+        counter_guess = await get_counter_guess(bot, ctx, team2)
+
+        prev_score = team1.score
+        result_text = await update_scores(team1, team2, answer, guess, counter_guess)
+        await display(ctx, team1, team2, prompt, max_score, text=result_text)
+
+        team1.rotate_psychic()
+
+        # Per official Wavelength rules - if you score 4 you get to go again
```
Comment (kelzha) [inline games/telewave/telewave.py:84]: Which rules?  This one doesn't say anything about this: https://www.ultraboardgames.com/wavelength/game-rules.php

File: games/telewave/telewave.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    return winners.get_players()
+
+
+async def play(bot, ctx, team1, team2):
+    max_score = config["max_score"]
+    going_again = False
+    while team1.score < max_score and team2.score < max_score:
+        await wait(ctx, going_again, team1)
+        prompt = get_prompt()
+        answer = random.randint(0, 100)
```
Comment (kelzha) [inline games/telewave/telewave.py:59]: decimals?

File: games/telewave/telewave_config.json [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+        ["Round","Pointy"],
+        ["Proof that God exists","Proof that God doesn't exist"],
+        ["Sad movie","Happy movie"],
+        ["Waste of time","Good use of time"],
+        ["Scary animal","Nice animal"],
+        ["Mainstream","Niche"],
+        ["Short lived","Long lived"],
+        ["Nobody does it","Everybody does it"],
+        ["Smells bad","Smells good"],
+        ["Star Wars","Star Trek"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:123]: remove

File: games/telewave/telewave_config.json [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+        ["Bad mouthfeel","Good mouthfeel"],
+        ["Illegal","Legal"],
+        ["Deep thought","Shallow thought"],
+        ["Bad school","Good school"],
+        ["Never on time","Always on time"],
+        ["Won't live to 100","Will live to 100"],
+        ["Bad Disney character","Good Disney character"],
+        ["Similar","Identical"],
+        ["Bad president","Good president"],
+        ["Weird","Strange"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:255]: remove

File: games/telewave/telewave_config.json [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+        ["Need","Want"],
+        ["Dry food","Wet food"],
+        ["Normal thing to own","Weird thing to own"],
+        ["Straight","Curvy"],
+        ["Bad person","Good person"],
+        ["80s","90s"],
+        ["Ethical to eat","Unethical to eat"],
+        ["Movie","Film"],
+        ["Optional","Mandatory"],
+        ["Underrated letter of the alphabet","Overrated letter of the alphabet"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:93]: remove

File: games/telewave/telewave_config.json [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+        ["Underrated","Overrated"],
+        ["Messy food","Clean food"],
+        ["Unethical","Ethical"],
+        ["Bad gift","Good gift"],
+        ["Unfashionable","Fashionable"],
+        ["Freedom fighter","Terrorist"],
+        ["Unforgiveable","Forgiveable"],
+        ["Failure","Masterpiece"],
+        ["Harmless","Harmful"],
+        ["Gryffindor","Slytherin"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:151]: remove?

File: games/telewave/telewave_config.json [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+        ["Worst year in history","Best year in history"],
+        ["Socialist","Capitalist"],
+        ["Little known fact","Well known fact"],
+        ["Stationary","Mobile"],
+        ["Local issue","Global issue"],
+        ["Talent","Skill"],
+        ["Worst era to time travel","Best era to time travel"],
+        ["The worst","The best"],
+        ["Small number","Large number"],
+        ["1","0"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:214]: remove

File: games/telewave/telewave_config.json [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+        ["Old fashioned","Avant garde"],
+        ["Ugly word","Beautiful word"],
```
Comment (kelzha) [inline games/telewave/telewave_config.json:216]: remove

File: games/telewave/telewave.py (hunk 2 above)
Comment (vsporeddy) [reply games/telewave/telewave.py:84]: It does (see below), but I missed one part - this only applies if you're losing. Will fix.

```
An important catch up rule

No matter how much a team is losing in Wavelength, there's always a chance to catch up. If the Psychic's team scores 4 points, and is still losing to the other team after that round's scoring, they immediately take another turn (with a different player as Psychic).

This means that a team down 9-0 could hypothetically take 3 turns in a row and win.
```

File: games/telewave/telewave.py (hunk 3 above)
Comment (vsporeddy) [reply games/telewave/telewave.py:59]: Thought it'd be simpler for answers/submissions to only be integers. The average of the guesses is still calculated as a decimal

**Comment by vsporeddy** [comment]
@kelzha can you just send me your revised prompt list so I can copy paste

File: games/telewave/telewave_config.json (hunk 7 above)
Comment (vsporeddy) [reply games/telewave/telewave_config.json:151]: nuh this one is great

File: games/telewave/telewave_config.json (hunk 6 above)
Comment (vsporeddy) [reply games/telewave/telewave_config.json:93]: I kinda like this one

**Pull request merged**